src/Main.java

```diff
@@ -1,5 +1,114 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        //task 1
+        System.out.println("//task1");
+        byte b = 15;
+        short s = 2045;
+        int i = 58674857;
+        long l = 85764983576L;
+        float f = 12.6F;
+        double d = 320.5;
+        System.out.println("Значение переменной b с типом byte равно " + b);
+        System.out.println("Значение переменной s с типом short равно " + s);
+        System.out.println("Значение переменной i с типом int равно " + i);
+        System.out.println("Значение переменной l с типом long равно " + l);
+        System.out.println("Значение переменной f с типом float равно " + f);
+        System.out.println("Значение переменной d с типом double равно " + d);
+
+        //task 2
+        System.out.println("//task2");
+        double a = 27.12;
+        long c = 987_678_965_549L;
+        float k = 2.786F;
+        int t = 569;
+        short h = -159;
+        int g = 27897;
+        byte r = 67;
+        System.out.println();
+
+        //task 3
+        System.out.println("//task3");
+        byte ma = 23;
+        byte ri = 27;
+        byte na = 30;
+        short total = 480;
+        int ii = (total / (ma + ri + na));
```

Comment:
```suggestion
        int ii = total / (ma + ri + na);
```
скобки лучше не писать лишние